devnet.sh

```diff
@@ -86,6 +86,18 @@ init_dusk_func() {
   network = "testnet"
   walletonly = "false"
   safecallbacklistener = "true"
+  # Timeout cfg for rpcBus
+  timeoutsendbidtx = 5
+  timeoutgetlastcommittee = 5
+  timeoutgetlastcertificate = 5
+  timeoutgetmempooltxsbysize = 4
+  timeoutgetlastblock = 5
+  timeoutgetcandidate = 5
+  timeoutclearwalletdatabase = 0
+  timeoutverifycandidateblock = 5
+  timeoutsendstaketx = 5
+  timeoutgetmempooltxs = 3
+  timeoutgetroundresults = 5
```

Comment:
I think we can use Set in our tests, it would be a little less cumbersome than updating method signatures on all required places.
https://github.com/spf13/viper#setting-overrides